lib/internal/process/pre_execution.js

```diff
@@ -735,7 +735,8 @@ function runEmbedderPreload() {
 function loadPreloadModules() {
   // For user code, we preload modules if `-r` is passed
   const preloadModules = getOptionValue('--require');
-  if (preloadModules && preloadModules.length > 0) {
+  const isOrcastrationProcess = getOptionValue('--test');
```

Comment:
I'm not sure if we should add the logic in this function. `loadPreloadModules()` is called by `setupUserModules()`, which already has an option for skipping the preloads. Maybe we should plumb another option from the `prepareMainThreadExecution()` call in `lib/internal/main/test_runner.js` down through `setupUserModules()`. It's a bit more code, but might be better for any future cases that also want to skip preloads.

I'll leave that up to you though.